src/main/java/com/aws/greengrass/shadowmanager/ShadowManager.java

```diff
@@ -239,166 +239,168 @@ protected void install(InstallConfig installConfig) {
         }
 
         inboundRateLimiter.clear();
+        config.lookupTopics(CONFIGURATION_CONFIG_KEY).subscribe((what, newv) -> {
+            if (what.equals(WhatHappened.timestampUpdated)) {
+                return;
+            }
+            if (installConfig.configureSynchronizeConfig
```

Comment:
Why do we even have the `installConfig` seems like it just gets built on the `install` and all the values are set to true and it is not changing based on anything.

Reply:
Also is there a ways to not use any of this conditionals at all?. What if we just call `configureSynchronization` without any conditional and the we have the method read the configuration by doing a lookup

```java
Topics configTopics = config.lookupTopics(CONFIGURATION_CONFIG_KEY, CONFIGURATION_SYNCHRONIZATION_TOPIC);
```

And decide what to do based on the value it currently holds?

Reply:
Yes, we'll need address those in later PRs.

Reply:
If we are going to do it later, lets add a TODO